Dockerfile

```diff
@@ -2,6 +2,8 @@ FROM --platform=$BUILDPLATFORM rust:1.64 AS buildbase
 RUN rustup target add wasm32-wasi
 WORKDIR /src
 
+RUN rustup target add wasm32-wasi
```

Comment:
This is already done in line 2, do we need it here as well?

Reply:
Sorry, your are right. I will remove that.